python/GangaDirac/Lib/Backends/DiracBase.py

```diff
@@ -974,8 +957,26 @@ def updateMonitoringInformation(jobs_):
         #logger.debug('Monitor jobs    : ' + repr([j.fqid for j in monitor_jobs]))
         #logger.debug('Requeue jobs    : ' + repr([j.fqid for j in requeue_jobs]))
 
+        # Split all the monitorable jobs into groups based on the
+        # credential used to communicate with DIRAC
```

Comment:
Isn't each proxy for each group an individual credential object? In which case I agree with the `TODO` this should be more generic as can't any tools using the VOMS tools potentially have multiple groups for 1 user.